components/leadzen_ai/actions/unlock-email-details/unlock-email-details.mjs

```diff
@@ -0,0 +1,55 @@
+import app from "../../leadzen_ai.app.mjs";
+
+export default {
+  key: "leadzen_ai-unlock-email-details",
+  name: "Unlock Email Details",
+  description: "Unlocks detailed information about the specified email.",
```

Comment:
This doesn't seem to describe what the action does. The user is meant to provide a URL instead of an email. Also, the doc link is missing.
```suggestion
  description: "Fetches the work email of a LinkedIn profile based on the specified URL. [See the documentation](https://api.leadzen.ai/docs#/People/people_search_work_email_api_people_linkedin_url_work_email_post)",
```